docs/contributing/getting-started-developers.md

```diff
@@ -9,103 +10,120 @@ This document covers:
 - [How to run the extension locally](#how-to-run-the-extension-locally)
 - [How to develop the client](#how-to-develop-the-client)
 - [How to built a local vsix](#how-to-built-a-local-vsix)
+- [Tests](#tests)
+- [Under the hood](#under-the-hood)
 
 Before starting make sure you read the [Web Template Studio arquitecture](application-architecture.md) document.
 
-If you just want to take advantage from Web Template Studio extension, check the [Web Template Studio Installation Instructions page](../install.md).
+If you just want to take advantage from *Web Template Studio* extension, check the [Web Template Studio Installation Instructions page](../install.md).
 
 ## Prerequisites
 
 1. Install [Node.js](https://nodejs.org/en/download/)
-2. Install [Gulp](https://gulpjs.com/)
-3. Install [Git](https://git-scm.com/downloads)
-4. Install [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1)
-5. Install [Visual Studio Code](https://code.visualstudio.com/)
-6. Install [Yarn](https://yarnpkg.com/en/docs/install). You will need to finish installing Node.js before you install Yarn.
-7. Install [Typescript](https://www.typescriptlang.org/) globally. `npm install -g typescript` 
-8. Install [React-scripts]() globally `npm install -g react-scripts`
-9. Run the command `npm config set scripts-prepend-node-path true`. This tells VSCode which Node version to run during the extension compilation (otherwise you'll get an error during the build process).
+1. Install [Gulp](https://gulpjs.com/)
+1. Install [Git](https://git-scm.com/downloads)
+1. Install [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1)
```

Comment:
Counter is showing 1. on all items

Reply:
This is my preferred way to use lists in markdown and seems that github is ok with it https://guides.github.com/features/mastering-markdown/
If you check the preview of the file, it works ok, but when you create the list you don´t bother about how many items or anything, and just put 1. in all of them

Can change if you don´t like this approach :) I just find it easier to manage when you change/add/remove items from the list.

Reply:
Much easier! Wasn't aware of this possibility